tests/e2e/cypress/integration/index.spec.js

```diff
@@ -1,25 +1,44 @@
 describe("Index page spec", () => {
-    it("Gives user transit provider options", () => {
-        cy.visit("/")
-
-        cy.contains("Choose your transit provider")
-          .siblings(".btn")
-          .should('not.be.empty')
-          .each(($e) => {
-              expect($e).attr("href").to.match(/\/[a-z]{3,}$/)
-          })
-    })
-
-    it("Takes user to transit provider page", () => {
-        cy.visit("/")
-
-        cy.get(".buttons .btn")
-          .first()
-          .click()
-
-        cy.contains("Let’s do it!")
-          .then(($e) => {
-            expect($e).attr("href").to.match(/\/eligibility\/$/)
-          })
-    })
+  it("Gives user transit provider options", () => {
+    cy.visit("/")
+
+    cy.contains("Choose your transit provider")
+      .siblings(".btn")
+      .should('not.be.empty')
+      .each(($e) => {
+        expect($e).attr("href").to.match(/\/[a-z]{3,}$/)
+      })
+  })
+
+  it("Takes user to transit provider page", () => {
+    cy.visit("/")
+
+    cy.get(".buttons .btn")
+      .first()
+      .click()
+
+    cy.contains("Let’s do it!")
+      .then(($e) => {
+        expect($e).attr("href").to.match(/\/eligibility\/$/)
+      })
+  })
+
+  it("Has a payment options page", () => {
+    cy.visit("/")
+
+    cy.contains("Payment Options")
+      .click()
+
+    cy.contains("The contactless symbol is four curved lines")
+  })
+
+  it("Has a help page", () => {
+    cy.visit("/")
+
+    cy.contains("Help")
+      .click()
+
+    cy.contains("If you need assistance with the site, please reach out to the customer service team for your local transit provider.")
```

Comment:
This will be trickier, but on this page we should test that the contact info for the configured agency (or both if coming from the top-level index) is shown.

Reply:
To be resolved in different PR.

Reply:
Was resolved in #176
- checked that all info is present if coming from top-level index [(line 35 of help.spec.js)](https://github.com/cal-itp/benefits/pull/176/files?diff=unified&w=0#diff-b092ffc585b11e5dcca602107a22ae7f70a62183073a31e62a2fb02e9185bd7dR35)
- checked for info if coming from transit-agency-specific context [(line 45 of help.spec.js)](https://github.com/cal-itp/benefits/pull/176/files?diff=unified&w=0#diff-b092ffc585b11e5dcca602107a22ae7f70a62183073a31e62a2fb02e9185bd7dR45)